### DISCUSSION TITLE
Display order of cameras on Web UI main page

### DISCUSSION THREAD

**Discussion by brewtangclan** [blakeblackshear/frigate#813] [Ideas]
Is there any way to arbitrarily order the cameras on the Cameras page of the Web UI? I currently have three cameras, "back_patio" (16x9), "doorbell" (3x4), and "driveway" (16x9). At desktop resolutions, this results in the back patio and doorbell cameras showing on the first row, with a lot of white space between the back patio camera image and the driveway one below it.

Adding either a "display_order" or "display_name" (instead using this name for the title and display order in the web UI) key to the YAML would do the trick.

I know I could just rename them so they sort alphabetically in the order I want, but I'd prefer not to, as I have some automations that require my current naming conventions.

Much thanks for all your work on this project so far! I've tried a few solutions for AI object detection, including writing my own, and this is by far the most accurate and robust system I've tried.

**Comment by paularmstrong** [comment]
Ordering is something that I would consider for a future release, but I'd do it via the web UI itself. Keep in mind that the UI is responsive and scalable, so different browser sizes will result in more or fewer columns appearing, so as noted, (currently) the only control right now would be configuring an order
